### ISSUE TITLE
Feature: use PostgreSQL format function to sanitize identifiers and literals

### ISSUE THREAD

**Issue by lyuboxa** [ConduitIO/conduit-connector-postgres#158]
### Feature description

Identifiers and literals are not fully sanitized or escaped. This can lead to a connector error. 

Use PostgreSQL format or quote_ident function sanitize these. It allows for a full query or an individual string.
https://www.postgresql.org/docs/current/functions-string.html#FUNCTIONS-STRING-FORMAT

**Comment by rauanmayemir** [comment]
Just experienced this by using the following config:
```
logrepl.publicationName: cnd.outbox
logrepl.slotName: cnd.outbox_jobs
```

Connector will crash because of sql syntax error. (for some reason, publication is actually created, so maybe it crashes on slot creation)

**Comment by rauanmayemir** [comment]
Nevermind, it was a different issue, addressed in #280.

**Comment by lyuboxa** [comment]
> Just experienced this by using the following config:
> 
> ```
> logrepl.publicationName: cnd.outbox
> logrepl.slotName: cnd.outbox_jobs
> ```

Since replication slots cannot be created in a schema, I assume this is the name of the slot. As per PostgreSQL slots cannot contain dots. See https://www.postgresql.org/docs/current/warm-standby.html#STREAMING-REPLICATION-SLOTS-MANIPULATION

Seems like #280 address this, so I'm going to close this one.

**Comment by lyuboxa** [comment]
Ugh my bad, reopening, this is a general issue with how we escape identifiers.

**Comment by lyuboxa** [comment]
@rauanmayemir you are correct `cnd.outbox` should be fine since that is escaped in the create query.
